Products/CMFPlone/patterns/__init__.py

```diff
@@ -242,6 +241,8 @@ def tinymce(self):
 
         image_types = settings.image_objects or []
         folder_types = settings.contains_objects or []
+
+        site_path = generator.portal_url.replace(self.request['SERVER_URL'], '')
```

Comment:
something like (replace getSite with something from ``plone.app.layout.globals.portal``, which gets the root portal):
```
>>> from zope.components.hooks import getSite
>>> from urlparse import urlparse
>>> site_path = urlparse(getSite().absolute_path()).path
```
?